tools/packaging/kernel/build-kernel.sh

```diff
@@ -403,6 +403,8 @@ install_kata() {
 	# Install uncompressed kernel
 	if [ "${arch_target}" = "arm64" ]; then
 		install --mode 0644 -D "arch/${arch_target}/boot/Image" "${install_path}/${vmlinux}"
+	elif [ "${arch_target}" = "s390x" ]; then
+		install install --mode 0644 -D "arch/${arch_target}/boot/compressed/vmlinux" "${install_path}/${vmlinux}"
```

Comment:
double install?  can you reuse the previous (`arm64`) sentence ?

Reply:
- Removed double `install` and changed `arch_target` check to `s390` rather than `s390x` (oops)
- No, that file is not generated on s390x.